Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -67,31 +69,36 @@
     // and avgReview is a weighted average of the userRating and the charityNavigatory API rating
     // weights for the averages are stored as class constants 
     private double calcCharityTrendingScore(Charity charity) {
-        boolean hasCharityNavRating = false;
         double charityNavRating, scaledCharityNavRating, userRating, avgReview;
+        charityNavRating = charity.getCharityNavRating();
+        boolean hasCharityNavRating = true;
+        /* assume all charities have charityNavRating
         if (charity has CharityNavigator rating) {
             hasCharityNavRating = true;
-            //charityNavRating = GET request to db for CharityNavigatorAPI rating;
+            charityNavRating = GET request to db for CharityNavigatorAPI rating;
         }
+        */
         scaledCharityNavRating = charityNavToScale * charityNavRating;
         //userRating = GET request to db for charity star rating;
```

Comment:
it will good to move the logic specific to chairty navigator API in a separate function. Since in future you also plan to add google trend API. So in that case this func will be neat and just call two diff func and then take average and return.

Reply:
done